src/CabanaPD_Types.hpp

```diff
@@ -14,19 +14,49 @@
 
 namespace CabanaPD
 {
+// Mechanics types.
 struct Elastic
 {
 };
 struct Fracture
 {
 };
+
+// Thermal types.
+struct TemperatureIndependent
```

Comment:
I am puzzled about this addition. Did we have TemperatureDependent before heat transfer, and now are adding TemperatureIndependent?